inst/scripts/importPDB.R

```diff
@@ -0,0 +1,76 @@
+library("bio3d")
+library("Biostrings")
+install.packages("seqnir")
+
+readRDS(MT.RDS)
+
+
+for (genes in gene_list){
+    results <- list()
+    # Translate the gene to its aa sequence
+    targetAA <- AAString(paste(a.a. sequence of the gene, collapse="")
+
+    for (PDB-ID/chains matched to gene){
+        # vectors for the data.frame that will be created at the end of this for loop.
+        rowNames <- c()
+        resIDs <- c()
+        aaType <- c()
+        sidechainXYZ <- c()
+        calphaXYZ <- c()
+
+        # load the PDB file.
+        subject <- read.pdb(PDB-ID)
+        # get the aa sequence of the subject
+        subjectAA <- AAString(paste(PDB$seqres, collapse=""))
+        #Perform pairwisealignment..
+        ali <- pairwiseAlignment(subjectAA, targetAA)
+        # ^^^ Q: what should the gapExtension and gapOpening penalties be?
```

Comment:
Use default values for matrix / opening / extension. In this scenario you expect 100% identity...